README.md

```diff
@@ -425,6 +426,26 @@ suc:
   version: v0.10.0
 ```
 
+### ArgoCD
+
+The ArgoCD bundle deploys [ArgoCD](https://argo-cd.readthedocs.io/en/stable/).
+
+To configure the bundle, use the `argocd` block:
+
+```yaml
+#cloud-config
+
+# Specify the bundle to use
+bundles:
+  - targets:
+      - run://quay.io/kairos/community-bundles:argocd_latest
+
+# Specify argocd settings
+argocd:
+  values: ....
+  version: ...
```

Comment:
Maybe add a comment that this should be the helm chart version ([to be used here](https://github.com/kairos-io/community-bundles/pull/71/files#diff-f4e91c7fe727a258ae196caa38c00257ebbf507a36546601a756401fcee54442R17)) and not the app version?